test

```diff
@@ -214,6 +214,15 @@ function dep_pass {
 	fi
 }
 
+function errcheck_pass {
+	echo "Running errcheck tests..."
+        export PATH="$PATH:$GOPATH/bin"
+        if ! which errcheck > /dev/null ; then
+                go get github.com/kisielk/errcheck || { echo "Failed to install 'errcheck'"; exit 255; }
```

Comment:
Moved `go get` in `.travis.yml`, but I don't think this check should be located inside the `fmt_pass`. Using dedicated function you can run only `errcheck` or skip it.